server/sandbox_stop.go

```diff
@@ -24,10 +25,13 @@ func (s *Server) StopPodSandbox(ctx context.Context, req *pb.StopPodSandboxReque
 	if err != nil {
 		return nil, err
 	}
-
-	if err := s.netPlugin.TearDownPod(netnsPath, podNamespace, sb.id, podInfraContainer.Name()); err != nil {
-		return nil, fmt.Errorf("failed to destroy network for container %s in sandbox %s: %v",
-			podInfraContainer.Name(), sb.id, err)
+	if _, err := os.Stat(netnsPath); err == nil {
```

Comment:
mmm :confused: 

the thing is:

1. stat netns
2. if there's no error in stat'ing, call teardown
3. if there's any error in stat'ing other than "not found" error, report it
4. otherwise skip the whole thing

your code does:

1. if there's an error in stat'ing
2. if the error is "not found", exit the stop container routine (???)
3. otherwise return the error